src/listener.iced

```diff
@@ -103,10 +102,11 @@ exports.Listener = class Listener
 
   got_new_connection : (x) ->
     throw new Error "@got_new_connection() is pure virtual; please implement!"
-   
+
   ##-----------------------------------------
 
   _make_server : () ->
+    net = require 'net' + ''
```

Comment:
actually doing this dynamic require will make the packager skip it so we could move this back up